app/controllers/base.rb

```diff
@@ -61,7 +62,13 @@ def site
 				if ENV['SITE_PATH']
 					LocalJob.local_site(params['site'])
 				else
-					Tint::Site.new(Tint.db[:sites][site_id: params['site'].to_i])
+					users = Tint.db[:sites].join(:site_users, site_id: :site_id).
+					                        where(sites__site_id: params['site'].to_i)
+					site = users.first.merge(users: users.map { |u|
```

Comment:
It's a join of user and site -- so it contains both.  I could specifically extract only the columns coming from the users table, but the extras don't hurt anything any it make the line simpler.